datasetinsights/estimators/__init__.py

```diff
@@ -2,5 +2,9 @@
 from .deeplab import DeeplabV3
 from .densedepth import DenseDepth
 from .faster_rcnn import FasterRCNN
+from .vgg_slam import VGGSlam
+from .vgg_slam_c_s import VGGSlamCS
+from .vgg_slam_multi_objects import VGGSlamMO
 
-__all__ = [Estimator, DeeplabV3, DenseDepth, FasterRCNN]
+
+__all__ = [Estimator, DeeplabV3, DenseDepth, FasterRCNN, VGGSlam, VGGSlamCS, VGGSlamMO]
```

Comment:
**__all__** should be a list of strings instead of a list of class names.
https://docs.python.org/3/tutorial/modules.html#importing-from-a-package